### DISCUSSION TITLE
How to make "Update related record" workflow work?

### DISCUSSION THREAD

**Discussion by Gotfryd** [YetiForceCompany/YetiForceCRM#17332] [How to]
For this example I will use current git developer version as of Yeti [ver. 6.5.9]

I'd like to create workflow in Products module that will update "Description" field in Vendors module by manual trigger. Products has 1:M relation to Vendors. 

First I create new workflow in Products module
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/e53a94ae-b69b-425e-9ccb-db4bec53ac46)

I set manual trigger (doesn't matter it's just for testing)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/3b05d429-717e-4f03-a445-ec9d841c57a4)

I set conditions (also doesn't matter, just to make workflow possible to trigger)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/d7b4fb9a-21b7-41b8-858e-29da586c527a)

I choose "Update related record" task type because that's what I need
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/6a8af1d8-0efe-4e5f-8834-fb4596212a23)

I select Description field from Vendors (based on reference fields)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/7cf82265-5d01-4259-8934-606a57d53b89)

I put some raw text that sould be replaced with in description field
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/5ed35f49-37c8-475d-8e14-50b38f2c1933)

Now I trigger the workflow for some product that meets the conditions
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/0a9e96de-1a6e-4551-959e-9a0476360c05)

In the Vendor record that is attached to the product I triggered workflow from, description has not changed
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/102538667/2f3712ca-6258-440c-af8a-ee855447174d)

I bet I'm making some simple mistake so my question is why is the description field in vendor not updating to "test" ?

**Comment by deepti4666** [comment]
I have noticed if you change the Conditions from "All" and move it to "Any" - it works better

**Reply by bartlomiejwtb** [reply]
Thank you for reply. Seems like it's random, your method worked for me once but when I tried to change related record description again - it didn't work. No clue what affects it.

**Reply by deepti4666** [reply]
Yes, Mine also stops randomly.

**Reply by bartlomiejwtb** [reply]
Ok, I know what's going on. Related field is updated with cron I believe, every 15 minutes for me 🙂 (p.s. I'm the author of this post just forgot to switch account before creating discussion)
